docs/src/main/tut/index.md

```diff
@@ -34,11 +53,49 @@ Any http4s response can be streamed from an asynchronous source. http4s offers a
 of helpers to help you get your data out the door in the fastest way possible without
 tying up too many threads.
 
-{%code_ref ../test/scala/org/http4s/docs/AsynchronousExample.scala asynchronous_example %}
+```tut:silent
+// Make your model safe and streaming by using a scalaz-stream Process
+def getData(req: Request): Process[Task, String] = Process("one", "two", "three")
+
+val service = HttpService.apply {
+  // Wire your data into your service
+  case req@GET -> Root / "streaming" => Ok(getData(req))
+  // You can use helpers to send any type of data with an available EntityEncoder[T]
+  case GET -> Root / "synchronous" => Ok("This is good to go right now.")
+}
+```
 
 http4s is a forward-looking technology.  HTTP/2.0 and WebSockets will play a central role.
 
-{%code_ref ../../../examples/blaze/src/main/scala/com/example/http4s/blaze/BlazeWebSocketExample.scala blaze_websocket_example %}
+```tut:silent
```

Comment:
Regrettable code duplication with a runnable example.  My thinking is that this urge will lesson as this becomes more of a proper tutorial.